### PR TITLE
Create API documentation

### PR DESCRIPTION
**Pull request by vpetrusenko**

### CODE REVIEW

**Comment by gfedorenko** [comment]
Do we really need all of this dependencies and gems?

**Comment by vpetrusenko** [comment]
@gfedorenko i'm not sure now, maybe we should install all dependencies except of grape-swagger with yarn

File: Gemfile [hunk 1]
```diff
@@ -15,6 +15,10 @@ gem 'jquery-rails', '~> 4.3'
 gem 'kaminari',     '~> 1.1.1'
 gem 'doorkeeper',   '~> 4.2.6'
 gem 'grape'
+gem 'wine_bouncer'
```
Comment (gfedorenko) [inline Gemfile]: Set gem versions

**Comment by vpetrusenko** [comment]
@mod okay i will fix it today and will work with swagger-ui through yaml

**Pull request merged**